Meta/Lagom/Tools/CodeGenerators/LibWeb/WrapperGenerator/IDLParser.cpp

```diff
@@ -817,147 +820,150 @@ void resolve_function_typedefs(Interface& interface, FunctionType& function)
     resolve_parameters_typedefs(interface, function.parameters);
 }
 
-NonnullRefPtr<Interface> Parser::parse()
+Interface& Parser::parse()
 {
     auto this_module = Core::File::real_path_for(filename);
 
-    auto interface = make_ref_counted<Interface>();
-    interface->module_own_path = this_module;
-    s_resolved_imports.set(this_module, interface);
+    auto interface_ptr = make<Interface>();
+    auto& interface = *interface_ptr;
+    s_interfaces.set(move(interface_ptr));
```

Comment:
I suspect the app will have issues if the set() calls on s_interfaces return anything other than HashSetResult::InsertedNewEntry ? Is that ensured by the current logic, or is it worth a VERIFY() ?

Reply:
This is currently impossible, since `make<Interface>()` cannot return a `NonnullOwnPtr` with the same pointer than another existing one, so it will never return anything other than `AK::HashResult::InstertedNewEntry`. But it is still worth making it explicit through a `VERIFY()` in case this part gets refactored later. Fixed :)